tensorflow/python/data/kernel_tests/range_test.py

```diff
@@ -18,75 +18,106 @@
 from __future__ import print_function
 
 from absl.testing import parameterized
+import numpy as np
 
 from tensorflow.python.data.kernel_tests import test_base
 from tensorflow.python.data.ops import dataset_ops
 from tensorflow.python.framework import combinations
 from tensorflow.python.framework import errors
-from tensorflow.python.framework import test_util
 from tensorflow.python.framework import dtypes
 from tensorflow.python.platform import test
 
 
 class RangeTest(test_base.DatasetTestBase, parameterized.TestCase):
 
-  @combinations.generate(test_base.default_test_combinations())
-  def testStop(self):
-    dataset = dataset_ops.Dataset.range(5)
-    self.assertDatasetProduces(dataset, expected_output=range(5))
+  @combinations.generate(combinations.times(
+    test_base.eager_only_combinations(),
+    combinations.combine(output_type=[dtypes.int32, dtypes.int64, 
+        dtypes.float32, dtypes.float64, dtypes.string])))
```

Comment:
Does it make sense to be using `dtypes.string` for all of these examples if the start / stop / step values are integers?

What would your expected use of `tf.data.Dataset.range` with `output_type=dtypes.string` look like?

Reply:
removed dtypes.string